src/main/java/com/azure/cosmos/kafka/connect/CosmosDBConfig.java

```diff
@@ -67,6 +78,8 @@ public class CosmosDBConfig extends AbstractConfig {
     private String connKey;
     private String databaseName;
     private String providerName;
+    private boolean bulkModeEnabled;
+    private int maxRetryCount;
```

Comment:
Can these be final?

Reply:
updated to be final - also updated to final for databaseName, containerName etc